repos/system_upgrade/el7toel8/libraries/mounting.py

```diff
@@ -52,11 +52,9 @@ def __init__(self, target, **kwargs):
 
         def create(self):
             """ Create the isolation context """
-            pass
```

Comment:
Why are you removing those passes? I think that removal of those `pass` is mistake. If pylint tells you this is redundant pass, rather skip the check completely. There is really no need to remove those pass from my POV. (@oamg/developers ?)

Reply:
Because if we use only pass, we get these warning and lint will fail -  [W0107 unnecessary-pass] Unnecessary pass statement ...

We can skip these warning by add these on .pylintrc file if is more appropriate ...

Reply:
iiuc pass statement is unnecessary if you have a docstring, and that's precisely what unnecessary-pass check is about.
I have no string opinion on this check, so up to you @fellipeh how to handle that.

Reply:
Yes, that's the point. I think that we should skip this check completely.

Reply:
For me, it's ok to skip these check on `.pylintrc` file, as I said, we have a lot of these passes on our code..

So, if all of you it's ok to skip that, I'll do..